oops/gold_master/__init__.py

```
@@ -2325,28 +2313,17 @@ def write_summary(self, outdir):
 
         filepath = outdir / 'summary.py'
 
-        localpath = None
-        try:
-            localpath = str(filepath.retrieve(filepath))
-        except FileNotFoundError:
-            pass
-
-        if localpath:
+        if log_path.exists():
```

Ruff (0.15.12)

[error] Undefined name `log_path`

(F821)